ReleaseProcessAutomation/ReleaseProcessAutomation.csproj

```diff
@@ -1,17 +1,26 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <PackageId>ReleaseProcessAutomation</PackageId>
-        <Version>1.0.0</Version>
-        <Authors>re-motion</Authors>
-        <Company>RUBICON</Company>
         <OutputType>Exe</OutputType>
         <TargetFramework>net6.0</TargetFramework>
         <PackAsTool>true</PackAsTool>
         <ToolCommandName>rpa</ToolCommandName>
         <Nullable>enable</Nullable>
         <RootNamespace>ReleaseProcessAutomation</RootNamespace>
     </PropertyGroup>
+    
+    <PropertyGroup>
+        <PackageId>Remotion.ReleaseProcessAutomation</PackageId>
+        <Version>1.0.0</Version>
+        <Authors>rubicon IT GmbH</Authors>
+        <DevelopmentDependency>true</DevelopmentDependency>
+        <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+        <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+        <PackageProjectUrl>http://www.re-motion.org/</PackageProjectUrl>
+        <Description>The re-motion Release Process Automation tool provides automation of the release process with Jira, Git and MSBuild</Description>
+        <Copyright>Copyright (c) rubicon IT GmbH, www.rubicon.eu - Licensed under Apache License, Version 2.0</Copyright>
+        <PackageTags>re-motion ReleaseAutomation</PackageTags>
+    </PropertyGroup>
```

Comment:
If you check with the old project, most of the information was injected into the nuspec file and many was retrieved from the shared properties file in the build process. @a-ctor can probably help you with setting this up to work similiar to re-motion Framework or the other libraries.
```
    <id>$id$</id>
    <version>$version$</version>
    <title>$title$</title>
    <authors>$author$</authors>
    <owners>$author$</owners>
    <licenseUrl>http://opensource.org/licenses/Apache-2.0</licenseUrl>
    <projectUrl>http://www.re-motion.org</projectUrl>
    <description>$description$</description>
    <copyright>$copyright$</copyright>
```

Reply:
Yeah, problem is we still have the old build script and .NET Core project :/

For now I would leave them hard coded and when we have the project converted to the new build script we can fix this.

Reply:
I replaced all the things I could replace from the `Properties.props` and `Version.props`, but I'm not sure where all the other things are coming from or even how to add them if i knew where they would be coming from.

Reply:
The property chaining looks good now. I'm not sure I get the context of your question: "where the other things are coming from". I can't really answer this without interpreting my own vew of the world into your question so if you could either rephrase or we just talk :)